WebHostLib/static/styles/tracker__SMW.css

```diff
@@ -0,0 +1,146 @@
+@import url('https://fonts.googleapis.com/css2?family=Lexend+Deca:wght@100..900&display=swap');
+
+.tracker-container {
+    width: 500px;
+    box-sizing: border-box;
+    font-family: "Lexend Deca", Arial, Helvetica, sans-serif;
+    border: 2px solid black;
+    border-radius: 4px;
+    resize: both;
+
+    background-color: #000000;
+    color: white;
+}
+
+.hidden {
+    display: none !important;
```

Comment:
Please do not use `!important` in CSS. Prefer rule specificity.

Reply:
Was not in use (still there from the template). Is removed.